master/internal/api_runs.go

```diff
@@ -729,6 +732,11 @@ func archiveUnarchiveAction(ctx context.Context, archive bool, runIDs []int32,
 	for _, cand := range runCandidates {
 		visibleIDs.Insert(cand.ID)
 		switch {
+		case cand.ExpArchived:
+			results = append(results, &apiv1.RunActionResult{
+				Error: "Run is part of archived Search.",
```

Comment:
Would we be able to include what search it's part of?

Reply:
fixed it